pkg/controller/remotemachineset/remotemachineset_controller_test.go

```diff
@@ -841,7 +839,7 @@ func testMachineSet(name string, machineType string, unstompedAnnotation bool, r
 	}
 	// Add a pre-existing annotation which we will ensure remains in updated machinesets.
 	if unstompedAnnotation {
```

Comment:
This variable name needs to be adjusted. But this verification is probably not need for machinesets since the tests are already adding other labels.

Reply:
I actually screwed this up, this was supposed to stay as an Annotation test, it predates my work I just changed how it was initialized. Will fix up.

Reply:
Restored to it's original form.